services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+from django.db import transaction
+from db.models import Ticket, Order, User, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: dict,

```

Comment:
The `tickets` parameter is expected to be a dictionary, but it seems to be used as a list of dictionaries in the loop. Consider changing the type annotation to `list[dict]` to reflect its actual usage.